x/shield/types/types.go

```diff
@@ -33,15 +56,29 @@ func NewPool(shield sdk.Coins, totalCollateral sdk.Int, deposit MixedDecCoins, s
 	}
 }
 
+// Collateral records the collaterals provided by a provider in a shield pool.
 type Collateral struct {
-	PoolID            uint64             `json:"pool_id" yaml:"pool_id"`
-	Provider          sdk.AccAddress     `json:"provider" yaml:"provider"`
-	Amount            sdk.Int            `json:"amount" yaml:"amount"`
-	Withdrawing       sdk.Int            `json:"withdrawing" yaml:"withdrawing"`
-	TotalLocked       sdk.Int            `json:"total_locked" yaml:"total_locked"`
+	// PoolID is the id of the shield pool.
+	PoolID uint64 `json:"pool_id" yaml:"pool_id"`
+
+	// Provider is the chain address of the provider.
+	Provider sdk.AccAddress `json:"provider" yaml:"provider"`
+
+	// Amount is the collateral amount, excluding withdrawing or locked.
```

Comment:
including withdrawing and exclude locked

Reply:
done.